test/button/button-icon.visual-diff.js

```diff
@@ -73,14 +77,4 @@ describe('d2l-button-icon', function() {
 
 	});
 
-	const focus = async(page, selector) => {
```

Comment:
This is the non-generic code that we _think_ we'd end up writing a bunch if we don't have a generic solution.